src/pages/cli/core/plugins.md

```
@@ -36,12 +36,12 @@ mplx core plugins add <assetId> ./plugin.json
 ```
 The JSON file should contain the plugin configuration in the following format:
 ```json
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)